pkg/security/apis/security/types.go

```diff
@@ -42,6 +42,11 @@ type SecurityContextConstraints struct {
 	// of a VolumeSource (azureFile, configMap, emptyDir).  To allow all volumes you may use "*".
 	// To allow no volumes, set to ["none"].
 	Volumes []FSType
+	// AllowedFlexVolumes is a whitelist of allowed Flexvolumes.  Empty or nil indicates that all
```

Comment:
Here I'm talking about the volumes of type flex and not about structs. [In the proposal about Flexvolumes](https://github.com/kubernetes/community/blob/8eca356fcb06d336f5128d93a9f68ef3f3867286/contributors/devel/flexvolume.md) they called as "Flexvolumes" and I'm using the same name here.